src/clinical/api/clinical-api.ts

```diff
@@ -243,12 +241,13 @@ class ClinicalController {
 		const completionState = completionFilters[state] || {};
 
 		// FE filters digits out of search text for Donor search
-		const query: ClinicalDonorEntityQuery = {
-			programShortName,
+		const query: ClinicalDataDbQuery = {
```

Comment:
"Clinical Data Database Query" -> "Clinical Database Query" to avoid redundancy?